src/Component/Symbolizer/SLDRenderer/SLDRenderer.tsx

```diff
@@ -16,7 +16,7 @@ interface SLDRendererDefaultProps {
 export interface SLDRendererAdditonalProps extends Partial<SLDRendererDefaultProps> {
   wmsBaseUrl: string;
   layer: string;
-  rasterLayer: string;
+  rasterLayer?: string;
```

Comment:
Is this intended or just an artefact? If intended, it should be checked [here](https://github.com/terrestris/geostyler/pull/1000/files#diff-8a2ec6e31ba513cd599fb0bb89da0ebbR118) if `rasterLayer` is defined.